plugins/producer_plugin/include/eosio/producer_plugin/production_pause_vote_tracker.hpp

```diff
@@ -171,13 +180,18 @@ class production_pause_vote_tracker {
          }
       };
 
-      if (check != pause_check::producer) {
-         assert(check == pause_check::both || check == pause_check::other);
+      switch (check) {
+      case pause_check::both:
          process_vote_timing(latest_other_vote, status.other_vote);
-      }
-      if (check != pause_check::other) {
-         assert(check == pause_check::both || check == pause_check::producer);
+         // no break
```

Comment:
Could use `[[fallthrough]];`

Reply:
`error: 'fallthrough' attribute is only allowed on empty statements`

Reply:
Did you forget the semicolon at the end of it?

Anyway, it doesn't matter. It's not necessary.

Reply:
Yep. Fixed.